src/index.ts

```diff
@@ -13,6 +15,29 @@ const S = [0n, 1n, 2n, 4n, 8n, 16n];
 type Quadbin = bigint;
 type Tile = {x: number; y: number; z: number};
 
+const TILE_SIZE = 512;
+
+function quadbinToOffset(quadbin: bigint): [number, number, number] {
```

Comment:
Could you rename these 3 functions to be `cellTo...` and export them?

Reply:
Sure!

Reply:
I think you missed this comment

Reply:
Wops! You are totally right, let me amend this 🙏 

Too many things on the table 😆

Reply:
I think if we are going to add functions to the package we shouldn't change the names later. So best to include them as ` cellToOffset(cell: bigint)` etc, i.e. renaming also the parameter. It's not an issue for deck that the name changes

Reply:
Actually, scratch that. The convention in this package is to have `quadbin: Quadbin` as the parameter, so we should use that everywhere, including `cellToBoundary`

Reply:
What I'm afraid of is that we are proposing some changes that may require a **major version**, and I'm not sure if we should go for them in this PR, just it 🤷 

What's your proposal then? Mine is:

- Stick to `Quadbin` type everywhere
- Use `Quadbin` instead of `cell` in all the function names
- Release a major

Reply:
As stated originally, I want the API to be aligned with `quadbin-py`. Which it currently is. 
- Any new functions which are in `quadbin-py` should have the same name
- Any additional functions, like `cellToOffset` should follow the conventions already used

I see no value in renaming the existing function names, that would be a breaking change and require a major version and diverge from `quadbin-py`.

For the new functions we should stick with the conventions, so using `cell` in function names and `quadbin: Quadbin` in the parameters. It doesn't matter to the calling code what the name of the parameter is

Reply:
The API is modeled on https://github.com/uber/h3-js, which means that it is simpler to port code between the spatial indices, which is not the case if the functions are things like `quadbinToBoundary`

Reply:
> For the new functions we should stick with the conventions, so using cell in function names and quadbin: Quadbin in the parameters. It doesn't matter to the calling code what the name of the parameter is

Sounds like a plan, let's go for it 👍

Reply:
I'll say now, every function:

* Receives a `quadbin: Quadbin`
* Uses `cell` in its name

Could you please re-re-re-review it 😆 ?